src/include/partitioning/custom_part_lib.rb

```diff
@@ -293,13 +293,17 @@ def check_ok_fssize(size_k, volume)
 
       if Ops.less_than(size_k, min_size_k)
```

Comment:
http://www.rubydoc.info/github/yast/yast-ruby-bindings/Yast%2FOps.less_than

Claims that I should use > instead of Ops.less_than.

Anyone can confirm this?